testing/features/components/breadcrumbs.feature

```diff
@@ -4,6 +4,8 @@ Feature: Breadcrumbs component
   previous content pages they have traversed through
 
   Rule: Default Breadcrumb
+    This collapses down when viewing on small screen sizes (Mobile's e.t.c.)
```

Comment:
typo nitpick: this should be `(mobiles etc)`

Reply:
it should!